src/flashbar/collapsible-flashbar.tsx

```diff
@@ -0,0 +1,392 @@
+                      )
+                    : clsx(styles.flash, styles[`flash-type-${item.type ?? 'info'}`], styles.item)
+                }
+                ref={element => {
+                  if (isFlashbarStackExpanded) {
+                    expandedItemRefs.current[getAnimationElementId(item)] = element;
+                  } else {
+                    collapsedItemRefs.current[getAnimationElementId(item)] = element;
+                  }
+                }}
+                style={
+                  !isFlashbarStackExpanded || transitioning
+                    ? {
+                        [customCssProps.flashbarStackIndex]:
+                          (item as StackableItem).collapsedIndex ?? (item as StackableItem).expandedIndex ?? index,
+                      }
+                    : undefined
+                }
+                key={getItemId(item)}
+              >
+                {showInnerContent(item) && (
+                  <Flash
+                    // eslint-disable-next-line react/forbid-component-props
+                    className={clsx(
+                      animateFlash && styles['flash-with-motion'],
+                      isVisualRefresh && styles['flash-refresh']
+                    )}
+                    key={getItemId(item)}
+                    ref={shouldUseStandardAnimation(item, index) ? transitionRootElement : undefined}
+                    transitionState={shouldUseStandardAnimation(item, index) ? state : undefined}
+                    {...item}
+                  />
+                )}
+              </li>
+            )}
+          </Transition>
+        ))}
+      </TransitionGroup>
+    </ul>
+  );
+
+  return (
+    <div
+      {...baseProps}
+      className={clsx(
+        baseProps.className,
+        styles.flashbar,
+        styles[`breakpoint-${breakpoint}`],
+        styles.stack,
+        isFlashbarStackExpanded && styles.expanded,
+        getVisualContextClassname('flashbar')
+      )}
+      ref={mergedRef}
+    >
+      <>
+        {isFlashbarStackExpanded && renderList()}
+        {isCollapsible && (
+          <div
+            className={clsx(
+              styles.toggle,
+              isVisualRefresh && styles['visual-refresh'],
+              isFlashbarStackExpanded ? styles.expanded : styles.collapsed,
+              transitioning && styles['animation-running']
+            )}
+            onClick={toggleCollapseExpand}
+            ref={toggleElementRef}
+          >
+            <span className={styles.status} role="status">
+              {toggleButtonText && <h2 className={styles.text}>{toggleButtonText}</h2>}
+              <span className={styles['types-count']} id={itemCountElementId}>
+                <LiveRegion>
+                  {counterTypes
+                    .map(
+                      ({ type, labelName }: { type: FlashType; labelName: LabelName }) =>
+                        `${countByType[type]} ${(i18nStrings && i18nStrings[labelName]) ?? ''}`
```

Comment:
I think using this format makes assumptions about localization, i.e. that all locales put the number first and then the subject. There's two alternatives I can think of:

1) Format it with a colon like: `${countByType[type]}: ${i18nStrings[labelName]}`
2) Turn `labelName` into a function that customers can localize properly.

Reply:
Option 1 is more realistic at this point —but wouldn't it be rather `${i18nStrings[labelName]}: ${countByType[type]}`? (for example: "Error: 1", otherwise it would be "1: Error")

Reply:
Done as in previous comment.